CMakeLists.txt

```diff
@@ -97,8 +97,8 @@ endif()
 # for use by clang_complete, YouCompleteMe, etc.
 set(CMAKE_EXPORT_COMPILE_COMMANDS 1)
 
-if(NOT CUDAQ_LLVM_VERSION)
-  set(CUDAQ_LLVM_VERSION 16)
+if(NOT LLVM_VERSION_MAJOR)
```

Comment:
Is this going to interfere with the LLVM build variable of identical name in any way?

Reply:
See my other comment regarding LLVM variables as well. I think the value of this variable ultimately has to match the value you would get from LLVM when you load llvm options. I don't see a valid scenario where the value here can be different than the major version used to build our llvm dependencies.

Reply:
Point is this isn't strictly using the variable, it's setting it.